apps/data-types.md

```diff
@@ -256,6 +256,7 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | Attribute | Type | Description |
 | - | - |  - |
 | url | string | The URL of an image. |
+| isZoomEnabled | boolean | True if the product zoom is enabled for an image. |
```

Comment:
Indicates if the zoom is enabled for a product image.

Reply:
ok

Reply:
response state is now true or false